docs/cli.md

```diff
@@ -0,0 +1,114 @@
+---
+id: cli
+title: CLI
+---
+
+Run Prettier through the CLI with this script. Run it without any
+arguments to see the [options](./options.html).
```

Comment:
https://docusaurus.io/docs/en/doc-markdown.html#linking-other-documents

> This can help when you want to navigate through docs on GitHub since the links there will be functional links to other documents (still on GitHub), but the documents will have the correct html links when they get rendered.

`[options](./options.html)` -> `[options](options.md)`

Reply:
Ooh neat. I'll check if it works with the version we're using.

Reply:
I've tested, it works. 👍

Reply:
Done.